src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -139,6 +142,20 @@ Amplify.configure(outputs);
 
 </InlineFilter>
 
+<InlineFilter filters={["vue"]}>
+
+In your app's entry point, typically **main.ts** for Vue apps created using Vite, make the following edits:
```

Comment:
the previous example should show `main.ts`. Vite does not create a tsx file for non-vue projects

Reply:
👍